app/adapters/adapter.rb

```diff
@@ -33,16 +33,26 @@ class GitHubIssue
     def initialize(request)
       @logged_user = AuthorizeApiRequest.call(request.headers).result
       @client = Octokit::Client.new(access_token: @logged_user.access_token)
-      Octokit.auto_paginate = true
+      @client.auto_paginate = false
       @client
     end
 
     def get_github_user
       user_login = @client.user.login
     end
 
-    def list_issues(github_slug)
-      @client.list_issues(github_slug)
+    def list_issues(github_slug, page_number)
+      @client.list_issues(github_slug, page: page_number, per_page: 15)
+    end
+
+    def total_issues_pages(last_page)
+      header_response = @client.last_response.rels[:last]
+      if header_response.class != NilClass
```

Comment:
why are you using class here? `header_response != nil`should do the trick, right?

Reply:
Ok, corrected to `unless header_response.nil?`.